### ISSUE TITLE
Publish to the Central Repository?

### ISSUE THREAD

**Issue by l2dy** [XiaoMi/pegasus-java-client#107]
Have you considered publishing the components to Maven's [Central Repository](https://central.sonatype.org/)?

**Comment by neverchanje** [comment]
Sure. We are gonna publish a 1.11.10 package to GitHub maven repo: https://github.com/XiaoMi/pegasus-java-client/packages. Please wait.

**Comment by Shuo-Jia** [comment]
> Sure. We are gonna publish a 1.11.10 package to GitHub maven repo: https://github.com/XiaoMi/pegasus-java-client/packages. Please wait.

Now, you can get it from [https://github.com/XiaoMi/pegasus-java-client/packages](https://github.com/XiaoMi/pegasus-java-client/packages)

**Comment by l2dy** [comment]
The Central Repository is the default repository for Apache Maven, which makes it much easier to use than Maven packages on GitHub. Xiaomi has published several [com.xiaomi.infra](https://search.maven.org/search?q=com.xiaomi.infra) packages there, why not do the same for the Pegasus client?

**Comment by Shuo-Jia** [comment]
> The Central Repository is the default repository for Apache Maven, which makes it much easier to use than Maven packages on GitHub. Xiaomi has published several [com.xiaomi.infra](https://search.maven.org/search?q=com.xiaomi.infra) packages there, why not do the same for the Pegasus client?

Yeah, we are considering it, however, the [com.xiaomi.infra](https://search.maven.org/search?q=com.xiaomi.infra) packages need be confired because it has been for a long time.

**Comment by l2dy** [comment]
FYI, the group ID `com.xiaomi` was claimed by cuijianwei in https://issues.sonatype.org/browse/OSSRH-9928 on 05/20/14.

**Comment by Shuo-Jia** [comment]
> FYI, the group ID `com.xiaomi` was claimed by cuijianwei in https://issues.sonatype.org/browse/OSSRH-9928 on 05/20/14.

Yeah, thanks, we have already contacted him, too.
